thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java

```diff
@@ -109,7 +109,8 @@ public double getResponseRate() {
   }
 
   public int getTotalResponses() {
```

Comment:
For future developer, it is good if we can leave a comment, indicating what they expect to receive from getTotalResponses.

Reply:
added. thanks